src/test/java/com/softserveinc/ita/ProtocolTest.java

```diff
@@ -1,25 +1,49 @@
 package com.softserveinc.ita;
 
-import static com.codeborne.selenide.Selenide.webdriver;
-import static org.assertj.core.api.Assertions.assertThat;
-
 import com.softserveinc.ita.pageobjects.LoginPage;
+import com.softserveinc.ita.pageobjects.admin.ProtocolPage;
 import com.softserveinc.ita.pageobjects.util.TestRunner;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
+import static com.codeborne.selenide.Selenide.webdriver;
+import static com.softserveinc.ita.pageobjects.util.DataProvider.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
 public class ProtocolTest extends TestRunner {
 
-    @Test
-    public void verifyProtocolPageOpening() {
-        // TODO: 13.05.2022 use config files for credentials and expected url
+    @BeforeMethod
+    public ProtocolPage openProtocolPage() {
         new LoginPage()
-                .login("admin", "dtapi_admin")
+                .login(ADMIN_LOGIN, ADMIN_PASSWORD)
                 .openProtocolPage();
-        var expectedUrl = "https://dtapi.if.ua/admin/protocol";
-        var currentUrl =  webdriver().object().getCurrentUrl();
+
+        return new ProtocolPage();
+    }
+
+    @Test
+    public void verifyProtocolPageOpening() {
+
+        var expectedUrl = PROTOCOL_PAGE_URL;
+        var currentUrl = webdriver().object().getCurrentUrl();
```

Comment:
you could create a util class for such operations with the browser: 'WindowTabHelper'
and create method 
```
public String getCurrentUrl() {
        return getAndCheckWebDriver().getCurrentUrl();
    }
```